main.rb

```diff
@@ -0,0 +1,49 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+  end
+
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, there is some debug content getting printed to the console when I run your app:

![image](https://github.com/lengushuru/OOP-school-library/assets/77069011/ed030a55-2eda-4ad0-bfff-e9edf7b75346)

Please make sure that you remove old `puts` and `prints` that were intended for testing purposes! 🤓 🔧

Reply:
Thanks for reminder I will work on them straight away 💯